packages/smart/deploy/020_renounce_amm_factory.ts

```diff
@@ -0,0 +1,23 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { DeployFunction } from "hardhat-deploy/types";
+import { isHttpNetworkConfig, makeSigner } from "../tasks";
+import { AMMFactory__factory } from "../typechain";
+
+const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
+  const { deployments } = hre;
+
+  if (!isHttpNetworkConfig(hre.network.config)) throw Error("Cannot deploy to non-HTTP network");
+
+  const signer = await makeSigner(hre);
+  const ammFactoryDeploy = await deployments.get("AMMFactory");
+
+  const ammFactory = AMMFactory__factory.connect(ammFactoryDeploy.address, signer);
+
+  // Burn ownership cause you never catch us ridin' dirty.
+  await ammFactory.renounceOwnership();
```

Comment:
What does ammfactory ownership do now? I doubt we want to renounce ownership for test deploys but if it's just fees then that's fine since we will just redeploy to change them. But if there are any methods we would want to call to debug...

Reply:
AMMFactory needs to trust market factories to prevent malicious market factories from draining reward pools. We could keep ownership to add more in the future but this models how we should handle it in production.